tests/test_script_queue.py

```diff
@@ -813,6 +816,12 @@ async def test_bin_script_state(self):
                     index=index,
                     exe_name="run_script_queue.py",
                     initial_state=initial_state,
+                    cmdline_args=(
+                        "--standard",
+                        self.standardpath,
+                        "--external",
+                        self.externalpath,
+                    ),
```

Comment:
My one worry about this change is we no longer test ScriptQueue's ability to find the standard ts_standardscripts and ts_externalscripts packages. Mind you, doing that in `test_bin_script_state` is really not the best place because it is a bit difficult to check the values that it chose (we'd have to make a remote and read the appropriate event). I suppose we can live with that, but I'd rather preserve some kind of test of that if we can afford to require that the ts_standardscripts and ts_externalscripts packages are installed.